src/components/calculator-component.js

```diff
@@ -0,0 +1,13 @@
+function calculator(id,buttons) {
```

Comment:
again, you don't need to pass buttons or buttonGroup as parameters, you can define it directly into the component!